pkgs/by-name/ru/rust-parallel/package.nix

```
@@ -0,0 +1,30 @@
+{
+  fetchFromGitHub,
+  lib,
+  rustPlatform,
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "rust-parallel";
+  version = "1.18.1";
+
+  src = fetchFromGitHub {
+    owner = "aaronriekenberg";
+    repo = "rust-parallel";
+    rev = "v${version}";
+    hash = "sha256-4f/JE8KWYDdLwx+bCSSbz0Cpfy/g3WIaRzqCvUix4t0=";
+  };
+
+  cargoHash = "sha256-bhwA2Acl10Rz5uRxJT+RagDZloeztM2eWJmkHV6Ib6c=";
+
+  doCheck = false; # tests hard code /bin/bash
```

```suggestion
  postPatch = ''
    substituteInPlace tests/dummy_shell.sh \
      --replace "/bin/bash" "${bash}/bin/bash"
  '';

  checkFlags = [
    "--skip=runs_echo_commands_dry_run"

    "--skip=runs_regex_command_with_dollar_signs"
    "--skip=runs_regex_from_command_line_args_nomatch_1"
    "--skip=runs_regex_from_input_file_badline_j1"
  ];
```

I don't know why disabling last three tests is required though, I think better disabling individual test(s) if possible. (unless that are major part of tests.) In this case without patch 8 out of 27. With patch above 4 tests are failing.

---

Thanks for that.  I added bash to the inputs